tests/self/se_time.c

```diff
@@ -157,3 +157,26 @@ void selftest_delay(void)
 }
 
 REGISTER_SELF_TEST(delay, se_manual, &selftest_delay)
+
+void selftest_latency(void)
```

Comment:
"latency" is not a meaningful name. What latency is measuring? 
I'd suggest something like "selftest_clock_latency".